Program.cs

```diff
@@ -1,7 +1,40 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter Password: ");
+         string Password = Console.ReadLine ();
+         if (check (Password) == "true") Console.WriteLine ("Strong password.");
+         else Console.WriteLine ("Not a strong password. \n" + check (Password)) ;
+      }
+      static string check (string pass) {

```

Comment:
You could use something like this as return type. **(bool isStrong, string errorMsg)** 
With this, you can print error message if the isStrong property is false.

Reply:
Changes are done.